src/handlers/graphical.rs

```diff
@@ -436,9 +436,23 @@ impl GraphicalReportHandler {
 
         let mut contexts = Vec::with_capacity(labels.len());
         for right in labels.iter().cloned() {
-            let right_conts = source
-                .read_span(right.inner(), self.context_lines, self.context_lines)
-                .map_err(|_| fmt::Error)?;
+            let right_conts =
+                match source.read_span(right.inner(), self.context_lines, self.context_lines) {
+                    Ok(cont) => cont,
+                    Err(err) => {
+                        writeln!(
+                            f,
+                            "  [{} `{}`: {:?}]",
```

Comment:
We should print the source name, at least. Possibly the span, as well.

Reply:
There is no name at this point. We only have a `SourceCode` trait object, which will give the name but only if the read_span succeeds.